src/test/cucumber/update.feature

```diff
@@ -18,7 +18,7 @@ Feature: Update
 			| kotlin    |
 			| scala     |
 		When I enter "sdk update"
-		Then I see "Adding new candidates(s): kotlin"
+		Then I see "Setting candidate list to: activator groovy kotlin scala"
```

Comment:
I do prefer the specific messages stating whether we add/remove candidates, and exactly what was added/removed. In reality this list is _very_ long and will be overwhelming feedback for the user.